bin/MadGraph5_aMCatNLO/Utilities/gridpack_helpers.sh

```diff
@@ -128,3 +128,46 @@ prepare_reweight () {
     done
     cd ..      
 }
+
+# Extract decay width in reweighting
+extract_width () {
+
+    isnlo=$1
+    wd=$2
+    cdir=$3
+    name=$4
+        
+    hasauto=$(grep "auto" ${cdir}/${name}_reweight_card.dat | egrep -v "#")
+    
+    if [[ ${hasauto} != "" ]]; then
+      echo "extract computed widths and rewrite reweight card"
+
+      if [ "$isnlo" -gt "0" ]; then
+        cp ${wd}/processtmp/Events/pilotrun/events.lhe.gz temp.lhe.gz
+        gzip -d temp.lhe.gz
+        ./MG5_aMC_v2_6_5/Template/LO/bin/internal/extract_banner-pl temp.lhe banner.txt
```

Comment:
Hi @kskovpen . Thanks for updating. I totally missed the mail. I suggest to make this generic so that it also works in future versions.

Reply:
Updated!